en/manual/engine/entity-component-model/usage.md

```diff
@@ -0,0 +1,109 @@
+# ECS Usage
+
+## Classes
+
+The three parts of "Entity Component System" map to the following classes:
+
+- Entity - [`Stride.Engine.Entity`](https://doc.stride3d.net/latest/en/api/Stride.Engine.Entity.html)
+- Component - [`Stride.Engine.EntityComponent`](https://doc.stride3d.net/latest/en/api/Stride.Engine.EntityComponent.html)
+- System - [`Stride.Engine.EntityProcessor`](https://doc.stride3d.net/latest/en/api/Stride.Engine.EntityProcessor.html)
+
+
+## Minimal Setup
+
+A component can be defined by deriving a class from `EntityComponent`.
+
+By adding the attribute `DefaultEntityComponentProcessor` to an `EntityComponent`,  
```

Comment:
Can you add a sentence below that you can manually add non-default processors via EntityManager?

Reply:
I... certainly can, now that I've learned about them :D

However, I added them way further down. That's because my idea was, most people searching for this and ending up on this page, probably just want to get it to work.

So, first a minimal example to get it to work - then below the optional advanced stuff, in order of how often it is probably used. So the non-default processors, are also almost at the end. Agree?

PS
Funny that this is possible. Because [that "most helpful forum post" I mentioned in the issue for this](https://forums.stride3d.net/t/registering-entityprocessors/1679/2) made it seem like this attribute was seen as _the_ way to do it :D But hey, that's from 2018, no wonder it's outdated.

Reply:
While the code sample you added is technically ok, that's not how you would call it.
The most common EntityManager implementation is [SceneInstance](https://github.com/stride3d/stride/blob/master/sources/engine/Stride.Engine/Engine/SceneInstance.cs) and I see in other places in the manual it's referenced from scripts like 
```
SceneSystem.SceneInstance.Processors.Add(entityProcessor);
```

Reply:
Replaced the code example with yours